importer/api/__init__.py

```diff
@@ -0,0 +1,95 @@
+"""
+Import OLX data into LORE.
+"""
+
+from glob import glob
+from datetime import datetime
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+import tarfile
+import zipfile
+
+from xbundle import XBundle, DESCRIPTOR_TAGS
+from lxml import etree
+
+from learningobjects.api import create_course, create_lox
+
+
+def import_course_from_file(filename, user_id):
+    """
+    Import OLX from .zip or tar.gz.
+
+    Args:
+        filename (unicode): Path to archive file (zip or .tar.gz)
+    Raises:
+        ValueError: Unable to find single path inside archive file.
+    Returns: None
+    """
+    tempdir = mkdtemp()
+    if filename.endswith(".tar.gz"):
+        course = tarfile.open(filename, "r")
+    elif filename.endswith(".zip"):
+        course = zipfile.ZipFile(filename, "r")
```

Comment:
Thanks. I checked out the StackOverflow page and the [code snippet](https://github.com/edx/edx-platform/blob/c08ccee45a5722755e6c47efda487dcd509001d0/openedx/core/lib/extract_tar.py). I'm assuming this isn't part of a "pip-installable" package, so I should just copy/paste the whole file into LORE. Or is there some kind of `mitodl/utils` package floating around with bits and bobs like this? If not, should we create one?